openslides_backend/action/actions/structure_level_list_of_speakers/update.py

```diff
@@ -32,27 +32,33 @@ def validate_instance(self, instance: Dict[str, Any]) -> None:
     def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         db_instance = self.datastore.get(
             fqid_from_collection_and_id(self.model.collection, instance["id"]),
-            ["list_of_speakers_id", "meeting_id", "remaining_time"],
+            ["list_of_speakers_id", "meeting_id", "remaining_time", "initial_time"],
         )
         if "initial_time" in instance:
             for field in ("current_start_time", "spoken_time"):
                 if field in instance:
                     raise ActionException(
                         f"Cannot set initial_time and {field} at the same time."
                     )
-            if self.datastore.exists(
-                "speaker",
-                And(
-                    FilterOperator("begin_time", "!=", None),
-                    FilterOperator("meeting_id", "=", db_instance["meeting_id"]),
-                    FilterOperator(
-                        "list_of_speakers_id", "=", db_instance["list_of_speakers_id"]
+            if (
+                self.datastore.exists(
+                    "speaker",
+                    And(
+                        FilterOperator("begin_time", "!=", None),
+                        FilterOperator("meeting_id", "=", db_instance["meeting_id"]),
+                        FilterOperator(
+                            "list_of_speakers_id",
+                            "=",
+                            db_instance["list_of_speakers_id"],
+                        ),
                     ),
-                ),
+                )
+                or db_instance["initial_time"] != db_instance["remaining_time"]
```

Comment:
Use this as first position in if, because if is resolved as True, there is no need to ask the database (much more expensive) anymore